sharethis.libraries.yml

```diff
@@ -25,3 +25,17 @@ drupal.sharethispickerexternal:
   dependencies:
       - core/jquery
 
+drupal.sharethispickerexternalbuttonsws:
+  version: VERSION
```

Comment:
remove the Version constant from here. It picks the version from drupal core version. Same applied for below